tests/models/trelloTest.js

```diff
@@ -1076,4 +1076,88 @@ exports.testModelTrello = {
       test.done();
     },
   },
+
+  /**
+   * handleCardLabelCreate method
+   */
+  handleCardLabelCreate : {
+
+    /**
+     * handleCardLabelCreate callApi
+     *
+     * @param test
+     */
+    callApi : function ( test ) {
+      test.expect( 2 );
+
+      var trello = new Trello( {} );
+      var dataTest = {
+        id : 'idTest',
+        card : {
+          id : 'idCard'
+        }
+      };
+
+      trello.callApi = function ( apiPath, method ) {
+        test.strictEqual( apiPath, '/cards/' + dataTest.card.id + '/idLabels/');
+        test.strictEqual( method, 'POST' );
+      };
+
+      trello.handleCardLabelCreate( dataTest, function () {
+      } );
+
+      test.done();
+    },
+
+    /**
+     * handleCardLabelCreate callbackSuccess
+     *
+     * @param test
+     */
+    callbackSuccess : function ( test ) {
+      test.expect( 2 );
+
+      var trello = new Trello( {} );
+
+      trello.callApi = function ( apiPath, method, data, callback ) {
+        callback( false, undefined );
+      };
+
+      trello.handleCardLabelCreate(
+        {
+        },
+        function ( error, response ) {
+          test.strictEqual( error, false );
+          test.strictEqual( response, undefined );
+        }
+      );
+
+      test.done();
+    },
+
+    /**
+     * handleCardLabelCreate callbackError
+     *
+     * @param test
+     */
+    callbackError : function ( test ) {
+      test.expect( 2 );
+
+      var trello = new Trello( {} );
+
+      trello.callApi = function ( apiPath, method, data, callback ) {
+        callback( true, { test : true } );
+      };
+
+      trello.handleCardLabelCreate(
+        {},
+        function ( error, response ) {
+          test.strictEqual( error, true );
+          test.strictEqual( response.test, true );
+        }
+      );
+
+      test.done();
+    },
+  },
 };
```

Comment:
Damn it. Okay. :+1: :)

Pedro Teixeira <notifications@github.com> schrieb am Mi., 18. Feb. 2015
22:37:

> In tests/models/trelloTest.js
> <https://github.com/natuelabs/porcupine/pull/4#discussion_r24943346>:
>
> > +
> > +      trello.callApi = function ( apiPath, method, data, callback ) {
> > +        callback( true, { test : true } );
> > +      };
> > +
> > +      trello.handleCardLabelCreate(
> > +        {},
> > +        function ( error, response ) {
> > +          test.strictEqual( error, true );
> > +          test.strictEqual( response.test, true );
> > +        }
> > +      );
> > +
> > +      test.done();
> > +    },
> > +  },
> >  };
>
> There is no new line at all files, jshint is not complaining too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/natuelabs/porcupine/pull/4/files#r24943346>.
>